packages/suite/src/hooks/wallet/useCoinmarketSpend.ts

```diff
@@ -218,7 +220,13 @@ export const useCoinmarketSpend = ({
                                             {
                                                 ...DEFAULT_PAYMENT,
                                                 address: trade.destinationAddress || '',
-                                                amount: trade.cryptoAmount?.toString() || '',
+                                                amount:
+                                                    trade.cryptoAmount && areSatsUsed
```

Comment:
`areSatsUsed` and `network.decimals` are missing in useEffect dependency array, please add them https://github.com/trezor/trezor-suite/pull/6173/files#diff-08473ff157cb97dce43c68a40172467d9cddb36d8b777981340b9757b65a43ecR256

Reply:
Thanks, missed the warning in IDE. Fixed.